projects/go-lib/src/lib/components/go-select/go-select.component.scss

```diff
@@ -174,9 +183,6 @@ body > .ng-dropdown-panel {
 
 // overrides for dark theme 
 .ng-select.go-select--dark {
-  .ng-dropdown-panel.ng-select-bottom {
-    border: 0;
-  }
```

Comment:
I'm not sure showing the white border is the route we want to take here. It's pretty bold. I looked through the DSM to see if we have dark selects documented anywhere and I couldn't find any. I reached out to @mobi/tangoe-ux-ui to see if they can provide some insight on what we should be doing here.

Reply:
Okay. Please let me know if is there any changes is required.

Reply:
@amitpatra91 I merged some SCSS changes to the select the other day and it caused merge conflicts with this branch. I went ahead and rebased with `dev` and fixed those conflicts for you and force pushed up the new changes.

These 3 lines actually ended up being added back, but we're still waiting to hear back from @mobi/tangoe-ux-ui on what route to take for styles here. When they get back with us we can proceed with this PR.

Alternatively, if you want to merge this as is and create an issue for the dark themed ng-select borders and handle it in a separate PR that would also work and allow us to merge these fixes with the next release.

Reply:
Per Teams:
> Amit:
> I will create separate pull request for go-select dark theme related changes. Please release the remaining css changes. Issue #755 created to address this.